cron/monthly/host-settlement.ts

```diff
@@ -90,11 +90,10 @@ export async function run(baseDate: Date | moment.Moment = defaultDate): Promise
 FROM "Transactions" as t
 INNER JOIN "TransactionSettlements" ts ON ts."TransactionGroup" = t."TransactionGroup" AND t.kind = ts.kind
 WHERE t."CollectiveId" = :CollectiveId
-AND t."createdAt" >= :startDate AND t."createdAt" < :endDate
 AND t."kind" IN ('PLATFORM_TIP_DEBT', 'HOST_FEE_SHARE_DEBT')
 AND t."isDebt" IS TRUE
 AND t."deletedAt" IS NULL
-AND ts."status" != 'SETTLED'`,
+AND ts."status" = 'OWED'`,
```

Comment:
Good change, this one could have resulted in duplicate settlements for the same transaction